app/_hub/kong-inc/grpc-gateway/index.md

```diff
@@ -0,0 +1,156 @@
+---
+name: gRPC-gateway
+publisher: Kong Inc.
+
+categories:
+  - transformations
+
+type: plugin
+
+desc: Access gRPC services through HTTP REST.
+description: |
+  A Kong plugin to allow access to a gRPC service via HTTP REST requests
+  and translate requests and responses in a JSON format. Similar to 
+  [gRPC-gateway].
+  
+source_url: https://github.com/Kong/kong-plugin-grpc-gateway
+
+license_type: MIT
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.1.x
+
+
+params:
+  name: grpc-gateway
+  route_id: true
+  protocols: ["http", "https"]
+  dbless_compatible: true
+
+  config:
+    - name: proto
+      required: false
+      default:
+      value_in_examples: path/to/hello.proto
+      description: |
+        Describes the gRPC types and methods,
+        [HTTP configuration](https://github.com/googleapis/googleapis/blob/fc37c47e70b83c1cc5cc1616c9a307c4303fe789/google/api/http.proto)
+        must be defined in the file.
+
+---
+
+## Purpose
+
+This plugin translates requests and responses between gRPC and HTTP REST.
+
+![grpc-gateway](https://docs.google.com/drawings/d/12hp4CPqrNPFhattL_cIoJptFvlAqm5wLQ0ggqI5mkCg/pub?w=749&h=370)
+
+Image credit: [grpc-gateway]
+
+## Usage
+
+This plugin should be enabled on a Kong `Route` that serves the `http(s)` protocol but proxies to a `Service` with the `grpc(s)` protocol.
+
+Sample configuration via declarative (YAML):
+
+```yaml
+_format_version: "1.1"
+services:
+- protocol: grpc
+  host: localhost
+  port: 9000
+  routes:
+  - protocols:
+    - http
+    paths:
+    - /
+    plugins:
+    - name: grpc-gateway
+      config:
+        proto: path/to/hello.proto
+```
+
+Same thing via the Admin API:
+
+```bash
+$ # add the gRPC service
+$ curl -XPOST localhost:8001/services \
+  --data name=grpc \
+  --data protocol=grpc \
+  --data host=localhost \
+  --data port=9000
+
+$ # add an http route
+$ curl -XPOST localhost:8001/services/grpc/routes \
+  --data protocols=http \
+  --data name=web-service \
+  --data paths[]=/
+
+$ # add the plugin to the route
+$ curl -XPOST localhost:8001/routes/web-service/plugins \
+  --data name=grpc-gateway
+```
+
+The proto file must contain the [HTTP REST to gRPC mapping rule](https://github.com/googleapis/googleapis/blob/fc37c47e70b83c1cc5cc1616c9a307c4303fe789/google/api/http.proto).
+
+In the example we use the following mapping (note the option (google.api.http) = {} section):
```

Comment:
```suggestion
The example uses the following mapping:
```

Reply:
Need to move the note (note the option (google.api.http) = {} section) below instead of in the intro